tasks/main.yml

```diff
@@ -1,8 +1,8 @@
 ---
 
 - name: Ensure that git is installed
-  apt: name=git
-       state=present
+  apt:
+    name: git
```

Comment:
Could all the dependencies installed on Debian only also be installed in a single with_items call?